src/pal/src/include/pal/context.h

```diff
@@ -486,7 +486,7 @@ inline static DWORD64 CONTEXTGetPC(LPCONTEXT pContext)
 #elif defined(_ARM64_) || defined(_ARM_)
     return pContext->Pc;
 #else
-#error don't know how to get the program counter for this architecture
+#error "don't know how to get the program counter for this architecture"
```

Comment:
It seems all of the changes to add quotes around the #error messages are not necessary.  The C++ standard says:
https://en.cppreference.com/w/cpp/preprocessor/error
(error_message can consist of several words not necessarily in quotes.)
gcc doesn't seem to have issue with that.
Could you please remove those changes and keep only those that are relevant to the intent of the change?

Reply:
GCC compiler doesn't like the ' in don't.

Reply:
Ah, I see. Ok then.